apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/class-wpcom-block-editor-nav-sidebar.php

```diff
@@ -85,5 +86,17 @@ public function enqueue_script_and_style() {
 			filemtime( plugin_dir_path( __FILE__ ) . $style_path )
 		);
 	}
+
+	/**
+	 * Point the dashboard link to wordpress.com in the editor sidebar for Gutenberg 14.5 compat
+	 *
+	 * @param array $settings Editor settings.
+	 * @return array Updated Editor settings.
+	 */
+	public function add_wpcom_dashboard_link( $settings ) {
+		$site_slug                               = preg_replace( '|^https?:\/\/|', '', home_url() );
+		$settings['__experimentalDashboardLink'] = 'https://wordpress.com/home/' . $site_slug;
+		return $settings;
+	}
```

Comment:
Thanks for getting this shipped and unblocking the Gutenberg release.

Now that we are not pressured for time as much, we should follow up to look at moving this setting to Jetpack. A couple reasons:
1. We need to figure out a good way for delivering these wpcom specific overrides from Jetpack since ETK is on feature freeze.
2. ETK can be disabled on AT sites, making this fix not present.
3. IIRC this nav sidebar in ETK is actually referring to the custom one we inject in the page/post editor. So it is a bit odd having this here since it supplies a general editor setting that influences the site editor via a separate custom sidebar plugin it does not related to.

cc @mattwiebe @agrullon95

Reply:
https://github.com/Automattic/jetpack/pull/27601 will do it